fxdc/fields.py

```
@@ -0,0 +1,81 @@
+import string
+from typing import Any, Generic, Optional, TypeVar
+
+from fxdc.exceptions import FieldError
+
+T = TypeVar("T")
+
+ACCEPTABLECHARACTERS = string.ascii_letters + string.digits + "_"
+
+
+class Field(Generic[T]):
+    def __init__(
```

Ruff (0.12.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)